docs/release_notes.md

```diff
@@ -5,6 +5,100 @@ hide:
+
+### Monitoring
+- [[PR #10982]](https://github.com/kubevirt/kubevirt/pull/10982) [machadovilaca] Refactor monitoring metrics
+- [[PR #10962]](https://github.com/kubevirt/kubevirt/pull/10962) [machadovilaca] Update monitoring file structure
+- [[PR #10853]](https://github.com/kubevirt/kubevirt/pull/10853) [machadovilaca] Refactor monitoring collectors
+- [[PR #10700]](https://github.com/kubevirt/kubevirt/pull/10700) [machadovilaca] Refactor monitoring alerts
+- [[PR #10693]](https://github.com/kubevirt/kubevirt/pull/10693) [machadovilaca] Remove MigrateVmiDiskTransferRateMetric
+- [[PR #10651]](https://github.com/kubevirt/kubevirt/pull/10651) [machadovilaca] Refactor monitoring  recording-rules
+- [[PR #10570]](https://github.com/kubevirt/kubevirt/pull/10570) [machadovilaca] Fix LowKVMNodesCount not firing
+- [[PR #10418]](https://github.com/kubevirt/kubevirt/pull/10418) [machadovilaca] Add total VMs created metric
+
+### Uncategorized
+- [[PR #11144]](https://github.com/kubevirt/kubevirt/pull/11144) [0xFelix] virtctl: Specifying size when creating a VM and using --volume-import to clone a PVC or a VolumeSnapshot is optional now
+- [[PR #11122]](https://github.com/kubevirt/kubevirt/pull/11122) [brianmcarey] Update runc dependency to v1.1.12
+- [[PR #11068]](https://github.com/kubevirt/kubevirt/pull/11068) [brianmcarey] Update container base image to use current stable debian 12 base
+- [[PR #11050]](https://github.com/kubevirt/kubevirt/pull/11050) [fossedihelm] restrict default cluster role to authenticated only users
+- [[PR #11047]](https://github.com/kubevirt/kubevirt/pull/11047) [jschintag] Fix potential crash when trying to list USB devices on host without any
+- [[PR #11001]](https://github.com/kubevirt/kubevirt/pull/11001) [fossedihelm] Allow `kubevirt.io:default` clusterRole to get,list kubevirts
+- [[PR #10918]](https://github.com/kubevirt/kubevirt/pull/10918) [orelmisan] VMClone: Emit an event in case restore creation fails
+- [[PR #10916]](https://github.com/kubevirt/kubevirt/pull/10916) [orelmisan] Fix the value of VMI `Status.GuestOSInfo.Version`
+- [[PR #10914]](https://github.com/kubevirt/kubevirt/pull/10914) [brianmcarey] KubeVirt is now built with go 1.21.5
+- [[PR #10898]](https://github.com/kubevirt/kubevirt/pull/10898) [matthewei] vmi status's guestOsInfo adds `Machine`
+- [[PR #10879]](https://github.com/kubevirt/kubevirt/pull/10879) [brianmcarey] Built with golang 1.20.12
+- [[PR #10863]](https://github.com/kubevirt/kubevirt/pull/10863) [dhiller] Remove year from generated code copyright
+- [[PR #10809]](https://github.com/kubevirt/kubevirt/pull/10809) [orelmisan] Source virt-launcher: Log migration info by default
+- [[PR #10787]](https://github.com/kubevirt/kubevirt/pull/10787) [matthewei] virtctl support to add template label and annotation filters
```

Comment:
If we have a client sig for virctl then this can go there

Reply:
We don't so I'll leave this here for now. 
I've addressed all other comments - thank you for the review!